diff --git a/ios/RNFBAnalytics/RNFBAnalyticsModule.h b/ios/RNFBAnalytics/RNFBAnalyticsModule.h
index 8f0268e224d97683d193066f493417f7e4890f2b..45a1d0b8c8fb25b7c6836caf903897b74154c382 100644
--- a/ios/RNFBAnalytics/RNFBAnalyticsModule.h
+++ b/ios/RNFBAnalytics/RNFBAnalyticsModule.h
@@ -17,6 +17,7 @@
 
 #import <Foundation/Foundation.h>
 
+@import RNFBApp;
 #import <React/RCTBridgeModule.h>
 
 @interface RNFBAnalyticsModule : NSObject <RCTBridgeModule>
